.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "scripts/shared/ovn-kubernetes"]
+	path = scripts/shared/ovn-kubernetes
```

Comment:
Hmm although its comfortable to just stick this in there, it seems wrong as it's not a script.

I'd create a new directory entirely like `3rd-party` or something under the root directory (or maybe simply `submodules`) and stick it there.

Then, in the Dockerfile for the base image we can have a directive to copy this to `/go/src/github.com/submariner-io/` or some other location (`${SHIPYARD_DIR}/submodules` can also work).